index.html

```diff
@@ -19,11 +19,80 @@
     <!-- This is where all of our awesome CSS code goes -->
     <link rel="stylesheet" href="style.css" />
   </head>
-  <body>
-    <!--
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+<header>
+  <ul class = "header">
+    <li>
+      <img class = "logo" src = "https://raw.githubusercontent.com/imd1005-web-development-winter-2023/assignment-02-NurpoHB/main/.images/Logo.png" alt = "CrimeChronicles">
+    </li>
+    <li>
+    <h1>Homi-site</h1>
+  </li>
+  <li>
+    <label for="about us"></label>
+
+    <select name="about us"></select>
+    <option></option>
```

Comment:
It would make more sense to have an `a` element as a link to the about us page, as opposed to putting this in a drop down